priv/static/docs/development/testnet/local.md

```diff
@@ -36,12 +36,13 @@ iex -S mix
 To be able to fund some addresses you can specify in the configuration which will be the addresses and the amount as genesis pool during the network initialization
 
 - Ensure a fresh start
-```
+```bash
 make clean
 ```
 
 - Restart the node with some configuration
-```
+```bash
+rm -rf ./_build
```

Comment:
Why you need this one ?

Reply:
Bash syntax highlighting in docsify.

Reply:
I mean the `rm-rf ./build`

Reply:
Yes it is there to ensure a fresh start to be able to fund some addresses you can specify in the configuration which will be the addresses and the amount as genesis pool during the network initialization. Few weeks ago I tried this and `make clean` was not enough I had to remove the build to solve the errors.

Reply:
If you face this issue, it's due to a configuration based on compilation instead of runtime; you can check this part to avoid this

Reply:
Yes it is in fact due to the configuration during compilation differs from configuration during runtime. This is why in order to do a fresh start you have to explicitly remove the previous compiled build and rebuild the project again with new configuration. This is for dev environment.

Reply:
Should I keep this command for posterity?

Reply:
You can move this to use runtime configuration call instead

Reply:
Fix https://github.com/archethic-foundation/archethic-node/pull/74 to be reviewed